codefresh.yml

```diff
@@ -22,28 +22,29 @@ steps:
       - make semver/show
       - make semver/export | tee -a ${{CF_VOLUME_PATH}}/env_vars_to_export
 
+  build_image:
+    title: Build image
+    type: build
+    description: Build docs-toolbox image
+    image_name: cloudposse/docs-toolbox
```

Comment:
Always name the image after the repo name. Otherwise we end up with too many names for the same thing.

Reply:
in this case `cloudposse/docs`